hack/tools.go

```diff
@@ -0,0 +1,8 @@
+// +build tools
+
+package main
+
+import (
+  // "go run github.com/gobuffalo/packr/packr" in Makefile
+  "github.com/gobuffalo/packr/packr"
```

Comment:
I think you need to change that to `_ "github.com/gobuffalo/packr/packr"` so it won't get erased on `go mod tidy`, because it's not actually being used in the code.

Reply:
actually I see that `github.com/gobuffalo/packr` is already being used in [here](https://github.com/argoproj-labs/argocd-autopilot/blob/main/util/assets/assets.go#L6). And it's already in `go.mod`. So that's not really the issue here.

Reply:
The required dependency is "github.com/gobuffalo/packr/packr" (double packr at the end).

Reply:
`_` added